src/components/com_tjdashboard/administrator/controllers/widget.php

```diff
@@ -42,7 +42,8 @@ public function getSupportedRenderers()
 			{
 				$rendererName = explode(".", $key);
 				$languageFilePath = JPATH_PLUGINS . '/tjdashboardrenderer/' . $rendererName[0];
-				$lang->load("plg_tjdashboardrenderer_" . $rendererName[0], $languageFilePath, null, false, true);
+				$lang->load("plg_tjdashboardrenderer_" . $rendererName[0], $languageFilePath, null, false, true)
```

Comment:
@monikaatvowel can you comment and explain the code change

Reply:
sure. done